xonsh/completers/python.py

```diff
@@ -206,11 +206,11 @@ def _safe_eval(expr, ctx):
     try:
         val = xonsh_safe_eval(expr, ctx, ctx, transform=False)
         _ctx = ctx
-    except:  # pylint:disable=bare-except
```

Comment:
I think that these were bare-excepts for a good reason.  We don't want a Ctrl+C or a raise SystemExit in a completer to take down xonsh